components/Terminal.js

```diff
@@ -153,21 +193,30 @@ class Terminal extends Component {
     ));
 
     return (
-      <div className="terminal-container-wrapper" style={{ color: this.props.color, ...this.props.style }}>
+      <div
+        className="terminal-container-wrapper"
+        style={{ color: this.props.color, ...this.props.style }}
+      >
         <Bar style={barColor} />
-        <div className="terminal-container terminal-container-main" style={backgroundColor}>
+        <div
+          className="terminal-container terminal-container-main"
+          style={backgroundColor}
+        >
           <div className="terminal-holder">
             <div className="terminal-content">
               <div className="terminal-input-area">
                 {output}
                 <p>
-                  <span className="terminal-prompt" style={prompt}>{this.state.prompt}</span>
+                  <span className="terminal-prompt" style={prompt}>
+                    {this.state.prompt}
+                  </span>
                   <input
                     className="terminal-main-input"
                     style={inputStyles}
                     type="text"
-                    ref={com => (this.com = com)}
+                    ref={com => this.com = com}
```

Comment:
Running `npm run lint` I don't get the any suggestion about this.
I will change the to older version of this line 👍